lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -1980,7 +1968,6 @@ private void createTasksCompletedEvents(List<TaskSummary> taskSummaries) {
     private int prio;
```

Comment:
without the setter you can make prio final. maybe you can do that for duration too? makes the whole object immutable.

Reply:
good point :)